test/types/bank-account.ts

```diff
@@ -0,0 +1,50 @@
+export default function bankAccount() {
+  const formEl = document.querySelector('form');
+
+  if (!formEl) return;
```

Comment:
Just noting on these early returns that the ts compiler is evaluating `formEl` as a _maybe-_`HTMLFormElement`, thus the early return escape hatch here will account for the code branching where `formEl` is `null`.